ui/app/templates/allocations/allocation/index.hbs

```diff
@@ -115,6 +115,66 @@
     </div>
   </div>
 
+  {{#if ports.length}}
+    <div class="boxed-section" data-test-allocation-ports>
+      <div class="boxed-section-head">
+        Ports
+      </div>
+      <div class="boxed-section-body is-full-bleed">
+        {{#list-table source=ports as |t|}}
+          {{#t.head}}
+            <th class="is-2">Name</th>
+            <th class="is-1">Dynamic?</th>
+            <th>Host Address</th>
+            <th>Mapped Port</th>
+          {{/t.head}}
+          {{#t.body as |row|}}
+            <tr data-test-allocation-port>
+              <td data-test-allocation-port-name>{{row.model.name}}</td>
+              <td data-test-allocation-port-is-dynamic>{{if row.model.isDynamic "Yes" "No"}}</td>
+              <td data-test-allocation-port-address>
+                <a href="http://{{network.ip}}:{{row.model.port}}" target="_blank" rel="noopener noreferrer">{{network.ip}}:{{row.model.port}}</a>
+              </td>
+              <td data-test-allocation-port-to>{{row.model.to}}</td>
+            </tr>
+          {{/t.body}}
+        {{/list-table}}
+      </div>
+    </div>
+  {{/if}}
+
+  {{#if services.length}}
+    <div class="boxed-section">
+      <div class="boxed-section-head">
+        Services
+      </div>
+      <div class="boxed-section-body is-full-bleed">
+        {{#list-table source=services as |t|}}
+          {{#t.head}}
+            <th class="is-2">Name</th>
+            <th class="is-1">Port</th>
+            <td>Tags</td>
+            <td>Connect?</td>
+            <td>Upstreams</td>
+          {{/t.head}}
+          {{#t.body as |row|}}
+            <tr data-test-service>
+              <td data-test-service-name>{{row.model.name}}</td>
+              <td data-test-service-port>{{row.model.portLabel}}</td>
+              <td data-test-service-tags>{{join ", " row.model.tags}}</td>
+              <td data-test-service-connect>{{if row.model.connect "Yes" "No"}}</td>
+              <td data-test-service-upstreams>
+                {{#each row.model.connect.SidecarService.Proxy.Upstreams as |upstream|}}
```

Comment:
More uppercase props. Is it possible to model these as fragments to benefit from the lowercasing that happens in the application serializer?

Reply:
Okay, I’ve done so. I made some judgment calls on naming, with these model names:

* `ConsulConnect`
* `SidecarService`
* `SidecarProxy`
* `SidecarProxyUpstream`